package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "near-wallet-selector",
-  "version": "8.5.4",
+  "version": "8.5.5",
```

Comment:
There's no need to bump the version on this PR, we will do it on the next release.